src/components/atoms/linkForEnum/LinkForEnum.tsx

```diff
@@ -17,7 +17,7 @@ type LinkForEnumProps = {
 export function LinkForEnum({ value }: LinkForEnumProps) {
   return (
     <Link
-      style={{ color: 'black', textDecoration: 'none' }}
+      style={{ textDecoration: 'none', color: 'inherit' }}
```

Comment:
If we set the `textDecoration and `color` in the `kurtTheme` in `theme.tsx` can the theme then be imported here instead of `inherit` here?
Seems odd to put a color in a theme file that then needs to be restated, but you're probably right, more just curious.

Reply:
Ya, I thought so, too.  I'm not sure if it's something about the `<Link>` or just `<a>` tags in general. But when I look at the computed styles in the console, I see that it receives the correct styling, but ignores it. No idea why.

Reply:
Super interesting. 
Feels silly with the ThemeProvider thing, but there must be something behind the scenes messing with it. 
Nice job figuring out that solution!